### ISSUE TITLE
macOS Monterey - Does this work or is there an alternative for this macOS release?

### ISSUE THREAD

**Issue by mcnaugha** [ololx/sbc-bitpool-expander#11]
Looks like Monterey really changes the way Bluetooth works in macOS. I can't figure it out. I presume you may need to explicitly state that this doesn't work with Monterey???

I fear that Apple may have done this on purpose for commercial reasons. Apple knows they need to offer something to improve Bluetooth access to their lossless and Hi-Res offerings in Apple Music. Perhaps higher bit rate SBC has the chance of making any new efforts look bad??? Perhaps they are intending on delivering adjustments to the SBC encoding that achieves the same thing and don't want the party spoiled. I'm hoping for a surprise this coming year where they will deliver a method to transmit ALAC losslessly. It may take abandoning Bluetooth for audio though. Maybe AirPods Pro 2 will feature "Wi-Fi Audio".

I'd like to see [SBC XQ](http://soundexpert.org/articles/-/blogs/audio-quality-of-sbc-xq-bluetooth-audio-codec) on Mac. The settings you access here are on the same route as SBC XQ.

Thanks.

**Comment by ololx** [comment]
Dear @mcnaugha,
Thank you for this issue. 
Currently, I do not know anything about the performance of the sbs bitpool expander on a macOS above the Mojave. I stay on macOS Mojave version 10.14.6.
I tried to find information about changes in bluetoothaudio work in macOs Monterey and it looks like you are right - apple has changed something in bluetooth utils. I will try to figure it out in the future.
At the moment, thanks to you, I have reflected this information in the readme in the "platform" badge like "platforn: 10.11|10.12|10.13|10.14".
I think we can close this issue or let it hang until better times?
